src/utils/network/profile-helper.js

```diff
@@ -0,0 +1,101 @@
+import {
+  buildThing,
+  getStringNoLocale,
+  getThing,
+  getWebIdDataset,
+  removeStringNoLocale,
+  saveSolidDatasetAt,
+  setThing
+} from '@inrupt/solid-client';
+import { RDF_PREDICATES } from '../../constants';
```

Comment:
I feel like this would be better set up like the model-helpers file. That way the project doesn't have several different styles of retrieving and interacting with data models.

Reply:
Do you mean to move `profile-helper` into `model-helpers` so they're all in one location? Think I could do that.